### PR TITLE
Added Backblaze S3 primary example

### PR DESCRIPTION
**Pull request by ChrisEdS**
Added an example configuration for Backblaze S3

### CODE REVIEW

**Comment by mmattel** [comment]
Cool 👍
Needs porting to next and 10.11
Question: what makes this example special so that it works with Backblaze?

**Comment by ChrisEdS** [comment]
In all other examples the port is defined - here we start with https:// and not port at the end of the endpoint url. I think that in general it does not hurt to have more examples.

**Comment by mmattel** [comment]
Is the missing port special to Backblaze?
I agree with the examples...

**Comment by ChrisEdS** [comment]
This is a working config - I'm not sure about the other examples. But Port 80 and http in general sounds outdated to me.

**Comment by mmattel** [comment]
Did you try https and 443?
Just for doublechecking.

File: modules/admin_manual/pages/configuration/files/external_storage/s3_compatible_object_storage_as_primary.adoc [hunk 1]
```diff
@@ -170,3 +170,38 @@ $CONFIG = [
     ],
 ];
 ----
+
+=== Backblaze S3
+
+The S3 backend can also be used to mount the bucket of a Backblaze S3 object store via the Amazon S3
+API into the virtual filesystem. The class to be used is `OCA\Files_Primary_S3\S3Storage`:
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/files/external_storage/s3_compatible_object_storage_as_primary.adoc:177]:
```suggestion
The S3 backend can also be used to mount the bucket of a Backblaze S3 object store via the Amazon S3 API into the virtual filesystem. The class to be used is `OCA\Files_Primary_S3\S3Storage`:
```
Just remove the linebreak

**Review by mmattel — APPROVED** [review]

**Pull request merged**